__tests__/app.test.js

```diff
@@ -33,29 +30,29 @@ describe("app", () => {
           expect(topics).toBeInstanceOf(Array);
         });
     });
-  });
-  it("200: GET - responds with an array of all the topic objects, with the correct properties", () => {
-    return request(app)
-      .get("/api/topics")
-      .expect(200)
-      .then(({ body }) => {
-        const { topics } = body;
-        topics.forEach((topic) => {
-          expect(topic).toMatchObject({
-            description: expect.any(String),
-            slug: expect.any(String),
+    it("200: GET - responds with an array of all the topic objects, with the correct properties", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          topics.forEach((topic) => {
+            expect(topic).toMatchObject({
+              description: expect.any(String),
+              slug: expect.any(String),
+            });
           });
         });
-      });
-  });
-  it("200: GET - responds with an array of all the topic objects, with the correct data", () => {
-    return request(app)
-      .get("/api/topics")
-      .expect(200)
-      .then(({ body }) => {
-        const { topics } = body;
-        expect(topics).toEqual(data.topicData);
-      });
+    });
+    it("200: GET - responds with an array of all the topic objects, with the correct data", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          expect(topics).toEqual(data.topicData);
+        });
```

Comment:
You don't need this test. Whilst is is valid now, this will break if in future you have a feature where you can post a new topic.

Reply:
No problem, thank you :)